Tasks/PublishCodeCoverageResultsV1/publishcodecoverageresults.ts

```diff
@@ -30,6 +30,9 @@ async function run() {
             tempFolder = resolvePathToSingleItem(workingDirectory, reportDirectory, true);
         }
 
+        //Warning for V1 deprecation
```

Comment:
Can we have this warning on line  no 10, just at the beginning of run method before any logs are printed?

Reply:
would this visual represent to the user when they plan to add it in classic definition? Please check on how we can show this in yml definition as well.